src/Result/inc/Result.h

```diff
@@ -0,0 +1,24 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file Result.h
+* @brief Result Object function declarations.
+*/
+////////////////////////////////////////////////////////////////////////////////
+#include "Parser.h"
+#include <string>
+
+#ifndef RESULT_H
+#define RESULT_H
+
+namespace calculator
+{
+    class Result
+    {
+        public:
+            Expression expression;
```

Comment:
Make all of the members private.

You should create a constructor for Result that will take set this. (Or you can use setters). This is called dependency injection, it is a crazy intimidating word, but it just means that we want to have our object members initialized and set.

There are frameworks for more complex dependency injection(Dune uses components), but you can just create a constructor here.

```cpp
Result(Expression expression, float answer) : expression_(expression), answer_(answer) {}
```